python-sdk/setup.cfg

```diff
@@ -1,10 +1,10 @@
 [metadata]
-name = forta_agent
+name = forta_bot
```

Comment:
this will change the python package name (and break a lot of dependent code). lets hold off on this for now

Reply:
reverted package name